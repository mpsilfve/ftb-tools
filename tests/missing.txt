Kokokokkkk kokookokook KOKOKOKKKOK kokokokkkok-koira kokokkoko-koiraansa 1909-1910 190:122:1
