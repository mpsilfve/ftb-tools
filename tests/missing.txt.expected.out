
"<Kokokokkkk>" Missing Abc
"<kokookokook>" Missing abc
"<KOKOKOKKKOK>" Missing ABC
"<kokokokkkok-koira>"
	 "kokokokkkok-koira" N Sg Nom abc Heur #2
"<kokokkoko-koiraansa>"
	 "kokokkoko-koira" N Sg Ill Px3 abc Heur #2
	 "kokokkoko-koira" N Sg Par Px3 abc Heur #2
	 "kokokkoko-koiras" N Pl Nom Px3 abc Heur #2
	 "kokokkoko-koiras" N Sg Gen Px3 abc Heur #2
	 "kokokkoko-koiras" N Sg Nom Px3 abc Heur #2
"<1909-1910>"
	 "1909-1910" Num Card noabc Heur #2
"<190:122:1>"
	 "190:122:1" Num Card noabc Heur #3
